### ISSUE TITLE
Special device does not exist -message

### ISSUE THREAD

**Issue by Perkolator** [ventoy/Ventoy#710]
When updating Ventoy on USB with the linux script, I get this message:

```
Upgrade operation is safe, all the data in the 1st partition (iso files and other) will be unchanged!

Update Ventoy  1.0.32 ===> 1.0.33   Continue? (y/n)y
mount: /home/user/ventoy-1.0.33/tmp_mnt: special device /dev/sda2 does not exist.

Update Ventoy to /dev/sda successfully finished.
```
Is the mount error message here intended or is something wrong with the script?

**Comment by Perkolator** [comment]
1. Yes, this is the output of `lsblk` for "sda" before and after Ventoy update (after update the partitions are not mounted automatically):

```
NAME                MAJ:MIN RM  SIZE RO TYPE  MOUNTPOINT
sda                   8:0    1 57,9G  0 disk  
├─sda1                8:1    1 57,8G  0 part  /media/user/Ventoy
└─sda2                8:2    1   32M  0 part  /media/user/VTOYEFI
```

2. Here's the list & update process output:

```
$ sudo ./Ventoy2Disk.sh -l /dev/sda
[sudo] password for user:     

**********************************************
      Ventoy: 1.0.33  x86_64
      longpanda admin@ventoy.net
      https://www.ventoy.net
**********************************************

Ventoy Version in Disk: 1.0.33
Disk Partition Style  : GPT
Secure Boot Support   : NO

$ sudo ./Ventoy2Disk.sh -u /dev/sda

**********************************************
      Ventoy: 1.0.33  x86_64
      longpanda admin@ventoy.net
      https://www.ventoy.net
**********************************************

Upgrade operation is safe, all the data in the 1st partition (iso files and other) will be unchanged!

Update Ventoy  1.0.33 ===> 1.0.33   Continue? (y/n)y
mount: /home/user/ventoy-1.0.33/tmp_mnt: special device /dev/sda2 does not exist.

Update Ventoy to /dev/sda successfully finished.
```

3. log.txt:

```
############# Ventoy2Disk -l /dev/sda [x86_64] ################
22.1.2021 02.10.41 +0000
root permission check ok ...
MODE=list FORCE= RESERVE_SPACE= RESERVE_SIZE_MB=0
hexdump test ok ...
mkexfatfs test ok ...
vtoyfat test ok ...
tool check success ...
check tool work ok
Ventoy version in /dev/sda2 is 1.0.33
############# Ventoy2Disk -u /dev/sda [x86_64] ################
22.1.2021 02.11.19 +0000
no need to decompress tools
root permission check ok ...
MODE=update FORCE= RESERVE_SPACE= RESERVE_SIZE_MB=0
hexdump test ok ...
mkexfatfs test ok ...
vtoyfat test ok ...
tool check success ...
check tool work ok
Trying to umount /media/user/Ventoy ...
Trying to umount /media/user/VTOYEFI ...
update ventoy ...
Ventoy version in /dev/sda2 is 1.0.33
This is GPT partition style ...
mounting part2 ....
mounting part2 success
umount part2 success
```

**Comment by Perkolator** [comment]
Did little debugging myself and the mount error happens in `VentoyWorker.sh` on line 449, in the first iteration of the for loop, the 2nd iteration does the mount properly.

If I add `sleep 2` before the loop, there's no mount error. So, either do that, or redirect the mount command to dev/null.

**Comment by ventoy** [comment]
I have fixed it in 1.0.34

**Comment by Perkolator** [comment]
Confirm fixed. Thank you.
